Docker/startozwdaemon.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+shopt -s nocaseglob
+if [ ! -c "$USB_PATH" ]
+then
+	echo "USB Path \"$USB_PATH\" does not exist or is not a Character Device"
+	exit 128
+else
+	CMD="-s $USB_PATH"
+fi
+
+if [ ! -z "$MQTT_SERVER" ]
+then
+	CMD+=" --mqtt-server $MQTT_SERVER"
+	if [ ! -z "$MQTT_PORT" ]
+	then
+		CMD+=" --mqtt-port $MQTT_PORT"
+	else
+		CMD+=" --mqtt-port 1883"
+	fi
+fi
+	
+if [ -z "$STOP_ON_FAILURE" ]
+then
+	CMD+=" --stop-on-failure"
+else 
+	if [[ $STOP_ON_FAILURE != "false" ]]
+	then
+		CMD+=" --stop-on-failure"
+	fi
+fi
+
+if [ ! -z "$MQTT_TLS" ]
+then
+	if [[ $MQTT_TLS == "true" ]]
+	then
+		CMD+=" --mqtt-tls"
+	fi
+fi
```

Comment:
For `STOP_ON_FAILURE`, all values are truthy except for `false`, for `MQTT_TLS` all values are falsey except for `true`, could be confusing?

Reply:
its the "default" that switches them around. 
I think stop-on-failure should be enabled for the container, where as mqtt-tls is optional... 

happy to handle the logic differently though